docs/docs/50-user-guide/60-reference-docs/80-webhook-receivers/dockerhub/index.md

```diff
@@ -0,0 +1,109 @@
+---
+sidebar_label: Docker Hub
+description: How to configure Docker Hub webhooks with Kargo for automated artifact discovery.
+---
+
+# Docker Hub Webhook Receiver
+
+The Docker Hub Webhook Receiver responds when container images or charts are
+pushed to Docker Hub repositories.
+
+When this happens, the receiver "refreshes" `Warehouse` resources subscribed to
+the corresponding Docker Hub repository.
+
+:::info
+"Refreshing" a `Warehouse` means enqueuing it for immediate reconciliation by
+the Kargo controller, which will attempt to discover new artifacts from all
+subscribed repositories.
+:::
+
+## Configuring the Receiver
+
+To enable webhook support for Docker Hub, you must configure a Kubernetes
+`Secret` and reference it in your `ProjectConfig`.
+
+The `Secret` must include a `secret` key in its data map. This value is used to
+generate a unique, hard-to-guess URL for the webhook receiver, providing basic
+protection against unauthorized requests. Because Docker Hub webhook payloads
+are **not** signed, Kargo uses this static token for basic validation.
+
+:::note
+The following command is suggested for generating a complex shared secret and
+encoding it for use in the `data` field:
+
+```shell
+openssl rand -base64 48 | tr -d '=+/' | head -c 32 | base64
+```
+
+:::
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: dh-wh-secret
+  namespace: kargo-demo
+data:
+  secret: <your-base64-encoded-token-here>
```

Comment:
The word "token" wasn't used anywhere up to this point. It's been referred to as a secret and a shared secret (incorrect). I know it must seem silly, but if we're not consistent with our language here it _will_ confuse someone.